static/app/styles/animations.tsx

```diff
@@ -124,3 +123,28 @@ export const alertHighlight = (priority: string, theme: Theme) => keyframes`
     border-color: ${theme.alert[priority].border};
   }
 `;
+
+export const shake = ({
+  steps = 50,
+  intensity = 3,
+}: {steps?: number; intensity?: number | [number, number]} = {}) => keyframes`
+${new Array(steps)
+  .fill(0)
+  .map((_, i) => {
+    const [intensityX, intensityY] = Array.isArray(intensity)
+      ? intensity
+      : [intensity, intensity];
+
+    const rand1 = Math.round(Math.random() * intensityX);
+    const rand2 = Math.round(Math.random() * intensityY);
+
+    return `${i * (99 / steps)}% {
+      transform: translate(${rand1}px, ${rand2}px);
+    }`;
+  })
+  .join('\n')}
+
+  100% {
+    transform: none;
+  }
+`;
```

Comment:
SOmetimes I feel like javascript is just getting harder and harder to read

Reply:
Yes